src/pages/_index/_components/what-is-wasmcloud/index.tsx

```diff
@@ -0,0 +1,194 @@
+    title: React.ReactNode;
+    description: React.ReactNode;
+    link?: string;
+    linkText?: React.ReactNode;
+  }>;
+}> = [
+  {
+    id: 'build',
+    image: '/img/pages/home/what-is-wasmcloud/build.svg',
+    title: 'Build',
+    features: [
+      {
+        title: 'Integrate with your existing ecosystem',
+        description: (
+          <>
+            wasmCloud has first-tier support for Kubernetes, VS Code, ArgoCD, Backstage, Chainguard,
+            and more.
+          </>
+        ),
+      },
+      {
+        title: 'Build Reusable Components',
+        description: (
+          <>
+            Using reusable components means that platform teams can build features that can be
+            leveraged across your entire organization.
+          </>
+        ),
+      },
+      {
+        title: 'Bring-your-own language(s) for polyglot apps',
+        description: (
+          <>
+            Build and run applications from components written in C, C#, C++, Rust, Python, Go, C#,
+            TypeScript and more.
+          </>
+        ),
+      },
+    ],
+  },
+  {
+    id: 'compose',
+    image: '/img/pages/home/what-is-wasmcloud/compose.svg',
+    title: 'Compose',
+    features: [
+      {
+        title: 'Combine components into applications',
+        description: (
+          <>
+            Create applications by connecting WebAssembly components, first-party platform
+            capabilities, or your own extensions.
+          </>
+        ),
+      },
+      {
+        title: 'Elastically extensible',
+        description: (
+          <>
+            Easily extend the wasmCloud platform at runtime with secure, plugin-based custom
+            capabilities—including your own custom interfaces.
+          </>
+        ),
+      },
+      {
+        title: 'Portable and pluggable',
+        description: (
+          <>
+            Universal interface abstractions built on WebAssembly Interface Types (WIT) maximize
+            application portability across dev, QA, test, and prod.
+          </>
+        ),
+      },
+    ],
+  },
+  {
+    id: 'run',
+    image: '/img/pages/home/what-is-wasmcloud/run.svg',
+    title: 'Run',
+    features: [
+      {
+        title: 'Seamless orchestration',
+        description: (
+          <>
+            wasmCloud enables distributed applications to seamlessly operate across any cloud,
+            Kubernetes cluster, datacenter, or edge.
+          </>
+        ),
+      },
+      {
+        title: 'Automatically elastic, reliably resilient',
+        description: (
+          <>
+            wasmCloud automatically scales your components vertically to meet real-time demand;
+            scaling horizontally is easy with our flexible, flat compute mesh.
+          </>
+        ),
+      },
+      {
+        title: 'Location Agnostic',
+        description: (
+          <>
+            With local-first routing and at-least-once delivery, wasmCloud delivers cross-region,
```

Comment:
```suggestion
            With local-first routing and at-most-once delivery, wasmCloud delivers cross-region,
```

Reply:
I have to double check this occasionally: https://docs.nats.io/reference/faq#does-nats-guarantee-message-delivery